dev/com.ibm.ws.transport.http/src/com/ibm/ws/http/netty/NettyChain.java

```diff
@@ -268,12 +272,27 @@ private void channelFutureHandler(ChannelFuture future) {
                 if (TraceComponent.isAnyTracingEnabled() && tc.isDebugEnabled()) {
                     Tr.debug(this, tc, "Channel failed to bind to port:  " + future.cause());
                 }
+
+                // Check if the exception is or was caused by UnresolvedAddressException
+                Throwable cause = future.cause();
+                boolean unresolvedAddress = false;
+
+            while (cause != null) {
```

Comment:
We can check if we can do this in the Netty TCP utils, since that doesn't currently throw Exceptions, we are putting it here for ease.